### ISSUE TITLE
Updating Getting Started guide

### ISSUE THREAD

**Issue by engelmi** [engelmi/hirte#599]
In https://github.com/containers/bluechi/pull/597 the RPM package for `bluechi` has been renamed to `bluechi-controller`. This change should be reflected in the getting started page (see [here](https://github.com/containers/bluechi/blob/main/doc/docs/getting_started.md)) as soon as the next release of `BlueChi` is out.

**Comment by mwperina** [comment]
`bluechi-controller` RPM is still providing `bluechi` to not break upgrades, so it doesn't need to be change immediately:

```shell
# dnf repoquery --provides bluechi-controller-0.6.0-1.el9
Last metadata expiration check: 0:01:58 ago on Mon 13 Nov 2023 04:33:58 PM CET.
bluechi = 0.6.0-1.el9
bluechi-controller = 0.6.0-1.el9
bluechi-controller(x86-64) = 0.6.0-1.el9
hirte = 0.6.0-1.el9

# dnf install bluechi
Last metadata expiration check: 0:02:06 ago on Mon 13 Nov 2023 04:33:58 PM CET.
Dependencies resolved.
==============================================================================================================================
 Package            Arch   Version          Repository                                                                   Size
==============================================================================================================================
Installing:
 bluechi-controller x86_64 0.7.0-0.202311131457.git11acd4e.el9
                                            copr:copr.fedorainfracloud.org:group_centos-automotive-sig:bluechi-snapshot  77 k
Installing weak dependencies:
 bluechi-selinux    x86_64 0.7.0-0.202311131457.git11acd4e.el9
                                            copr:copr.fedorainfracloud.org:group_centos-automotive-sig:bluechi-snapshot  29 k

Transaction Summary
==============================================================================================================================
Install  2 Packages

Total download size: 106 k
Installed size: 187 k
Is this ok [y/N]: n
Operation aborted.

# dnf install bluechi-controller
Last metadata expiration check: 0:02:15 ago on Mon 13 Nov 2023 04:33:58 PM CET.
Dependencies resolved.
==============================================================================================================================
 Package            Arch   Version          Repository                                                                   Size
==============================================================================================================================
Installing:
 bluechi-controller x86_64 0.7.0-0.202311131457.git11acd4e.el9
                                            copr:copr.fedorainfracloud.org:group_centos-automotive-sig:bluechi-snapshot  77 k
Installing weak dependencies:
 bluechi-selinux    x86_64 0.7.0-0.202311131457.git11acd4e.el9
                                            copr:copr.fedorainfracloud.org:group_centos-automotive-sig:bluechi-snapshot  29 k

Transaction Summary
==============================================================================================================================
Install  2 Packages

Total download size: 106 k
Installed size: 187 k
Is this ok [y/N]: n
Operation aborted.
```

**Comment by jwmelto** [comment]
I don't know if this needs to be a new issue or not; the title is sufficient, but the original description is very narrow in scope.

I tried to run the Single Note Setup on RHEL 9.4 without success.
1. `/run/bluechi` directory isn't automatically created
2. even if the directory is created manually, the socket still isn't created
3. `journalctl -u bluechi-controller` does not show "Waiting for connection requests" either on TCP or unix socket. I did note that the package installed was 0.9.0-1, and the guide says 0.10.0-0 ... something major changed?
4. `bluechictl status` shows the node is offline

**Comment by engelmi** [comment]
@jwmelto Yes, in [v0.10.0](https://github.com/eclipse-bluechi/bluechi/releases/tag/v0.10.0) we enhanced the support for unix domain sockets, including the `bluechi-controller` to create the `bluechi.sock` on its own if necessary. Previously, the socket was only created via socket activation from the systemd unit `bluechi-controller.socket`. For this, however, you need to enable that unit (its not by default).
In https://github.com/eclipse-bluechi/bluechi/pull/1009 we also enabled the controller to use both, TCP and UDS, in parallel as sources and decided to default to UDS locally. This works smooth only for v0.10.0 with the mentioned changes. 
Unfortunately, we don't have the means to version the documentation. Maybe we can add that? @pypingou 

Long story short: 
Please upgrade to v0.10.0. It was released yesterday, so it might still take a while till its easily available - you can follow [these instructions here](https://github.com/eclipse-bluechi/bluechi/blob/main/CONTRIBUTING.md#testing) to install the latest copr version.
Or use the following config to use TCP for the agent instead:
```ini
[bluechi-agent]
NodeName=<name>
ControllerHost=127.0.0.1
ControllerPort=842
```

**Comment by mkemel** [comment]
bluechi-10.2 Bodhi update is currently in `testing` status, and will become stable in 5 days. We have released 10.2 right after 10.1, and 10.2 update has obsoleted 10.1 update, otherwise it would have already been available.

Actually no need to install from copr in this case,
```
dnf --enablerepo epel-testing install bluechi-controller bluechi-agent bluechi-ctl
```
will do the trick.

**Comment by engelmi** [comment]
This issue is quite old and the documentation has been updated several times. Closing.
